README.md

```diff
@@ -301,8 +302,8 @@ It is not possible to use different credentials for additional Backup Storage Lo
 
 ### Prerequisites
 
-* Velero 1.6.0 or later
-* AWS plugin must be installed, either at install time, or by running `velero plugin add velero/velero-plugin-for-aws:v1.2.1`
+* Velero 1.7.0 or later
+* AWS plugin must be installed, either at install time, or by running `velero plugin add velero/velero-plugin-for-aws:v1.3.0`
```

Comment:
Same here, maybe we can use the same style as https://github.com/vmware-tanzu/velero-plugin-for-microsoft-azure/pull/107 ?